test/test_unit.jl

```diff
@@ -611,6 +803,75 @@ isdir(outdir) && rm(outdir, recursive=true)
     for orientation in orientations
       @test flux_hlle(u, u, orientation, equations) ≈ flux(u, orientation, equations)
     end
+
+    # Test HLL flux with min_max_speed_einfeldt
```

Comment:
Same here

Reply:
Addressed in 8dff75c549be5a97b1db6209eaae3a276cde956a.